### PR TITLE
App Services updates

### PR DESCRIPTION
**Pull request by epwinchell**
Jira: https://issues.redhat.com/browse/RHCLOUD-18349

Screen Shot 2022-04-26 at 2 54 51 PM

### CODE REVIEW

File: chrome/application-services-navigation.json [hunk 1]
```diff
@@ -10,90 +10,113 @@
...
-                    "href": "/application-services/acs"
+                    "appId": "dbaas",
+                    "title": "Database Access",
+                    "href": "/application-services/databases"
                 },
                 {
-                    "title": "Documentation",
-                    "isExternal": true,
-                    "href": "https://docs.openshift.com/acs/3.66/welcome/index.html"
+                    "appId": "dataScience",
```
Comment (Hyperkid123) [inline chrome/application-services-navigation.json]: This appId is invalid. Can you use the original one?

File: main.yml [hunk 2]
```diff
@@ -754,44 +754,66 @@ service-registry:
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry
 
+
+
+
 application-services:
-  title: Application Services
+  title: Application and Data Services
```
Comment (Hyperkid123) [inline main.yml]: All of these changes are not necessary

**Comment by tiffanynolan** [comment]
@epwinchell - Is there a link that I can use to take a look at a version of this on a qa site?

**Comment by Hyperkid123** [comment]
@tiffanynolan this will end up updating QA env. Let me resolve the conflicts and send you a link.

**Comment by tiffanynolan** [comment]
Just got confirmation from the Connectors team that the 'Connectors Instances" nav item should be using the beta label. I'm not sure if that is there already, but it's breaking news. :)

**Comment by Hyperkid123** [comment]
> Just got confirmation from the Connectors team that the 'Connectors Instances" nav item should be using the beta label. I'm not sure if that is there already, but it's breaking news. :)

@tiffanynolan I was just about to ask. And what about the `Streams for Apache Kafka` under the `Subscriptions`?

**Comment by tiffanynolan** [comment]
> > Just got confirmation from the Connectors team that the 'Connectors Instances" nav item should be using the beta label. I'm not sure if that is there already, but it's breaking news. :)
> 
> @tiffanynolan I was just about to ask. And what about the `Streams for Apache Kafka` under the `Subscriptions`?

All Kafka related items (Streams for Apache Kafka) should be GA at this point.

File: chrome/application-services-navigation.json [hunk 3]
```diff
@@ -1,135 +1,126 @@
...
+        }
+      ]
+    },
+    {
+      "groupId": "data-services",
+      "title": "Data Services",
+      "icon": "database",
+      "navItems": [
         {
-            "title": "Advanced Cluster Security",
```
Comment (sachaudh) [inline chrome/application-services-navigation.json:94]: Can we add this one back as well? Noticed the navigation looking odd today

**Pull request merged**